docs/Commands.md

```diff
@@ -2,11 +2,131 @@
 
 Commands are used as shortcuts for long messages. If a message starts with the "trigger" then the message will be replaced with the Command.
 
-#### Example
+## Built-in commands
+
+Chatterino comes with a collection of built-in commands to help with channel management, Twitch interaction, and other misc. features.
+
+### `/block` & `/unblock`
+
+Usage: `/(block|unblock) <user>` 
+
+Blocks or unblocks the specified user. Blocking will hide that user's messages/whispers as well as disassociate them from you on Twitch. For more information about blocking, see [Twitch's help article](https://help.twitch.tv/s/article/how-to-manage-harassment-in-chat?language=en_US#BlockWhispersandMessagesfromStrangers).
+
+### `/chatters`
+
+Usage: `/chatters`
+
+Shows the total amount of chatters currently connected to the channel.
+
+### `/clearmessages`
+
+Usage: `/clearmessages`
+
+Clears all messages from the current split/channel. This command is purely visual and is not related to the clearchat moderator command. 
+
+### `/clip`
+
+Usage: `/clip`
+
+Creates a clip from the last 30 seconds of the current channel. Also provides an edit link to edit a clip from the current point in time.
+
+### `/debug-args`
+
+Usage: `/debug-args`
+
+Displays the arguments that chatterino was launched with.
+
+### `/delete`
+
+Usage: `/delete <msg-id>`
+
+Moderator only, deletes a message in chat with the provided ID. Usually used in mod actions as they can pass through the message ID.
+
+### `/fakemsg`
+
+Usage: `/fakemsg <raw message>`
+
+Displays the provided IRC messsage in chat as if it was sent from Twitch's IRC server.
+
+### `/ignore` & `/uningore`
+
+Usage: `/(ignore|unignore) <user>`
+
+Ignores or unignores messages for the specified user on Twitch. Ignoring will hide that user's messages/whispers. For more information about ignoring, see [Twitch's help article](https://help.twitch.tv/s/article/how-to-manage-harassment-in-chat?language=en_US#UseTheIgnoreFeature).
+
```

Comment:
```suggestion
```
Commands exist in Chatterino but state they are to be removed soon ™️ - [PH insert-deprecation-PR-here]

2nd note: As I see you've included a separate section of the help page for this entry, it appears while this logic is correct in browser chat, since we handle the loading of blocked users in-client with a drop-down setting, utilizing a separate "use ignore to still see their messages as moderator" is not the correct logic to inform people of. 
(since we just block them no matter what command is used)

Reply:
Cheers, removed!